styles/main.css

```diff
@@ -91,139 +91,76 @@ a {
     top: 5px;
 }
 
-#nav a {
-    color: #fff;
+a {
+    color: #897F7B;
 }
 
-#nav a:hover {
-    color: #D6E8D9;
+a:hover {
+    color: #E94E1B;
 }
     
 #about-page .active,
-#illustration-page .active,
+#work-page .active,
```

Comment:
you don't need `#about-page` and `#work-page`. you can just use the `.active` for all. no need to over specify